onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -1880,7 +1893,7 @@ static bool CastNeedFallbackToCPU(const onnxruntime::Node& node) {
 }
 
 std::unique_ptr<onnxruntime::IDataTransfer> CUDAExecutionProvider::GetDataTransfer() const {
-  return onnxruntime::make_unique<onnxruntime::GPUDataTransfer>(do_copy_in_default_stream_);
+  return onnxruntime::make_unique<onnxruntime::GPUDataTransfer>(this, do_copy_in_default_stream_);
```

Comment:
>this, do_copy_in_default_stream_ [](start = 64, length = 32)

just pass in EP? The option do_copy_in_default_stream could be accessed from EP.